src/main/java/dread/DreadAnalyzer.java

```diff
@@ -86,6 +86,7 @@ protected Function functionAt(Program program, Address addr) {
 		required.put("__cxa_guard_acquire", functionAt(program, "0x71011f3000"));
 		required.put("__cxa_guard_release", functionAt(program, "0x71011f3010"));
 		required.put("ReadConfigValue", functionAt(program, "0x71000003d4"));
+		required.put("AddEnumValue", functionAt(program, "0x71000148b8"));
```

Comment:
this will break analysis on non-enum classes, recommend overriding in your child class to include this func

Reply:
oh, or move over to the known functions analyzer, since it looks like you're using it more for that